torch/cuda/nvtx.py

```diff
@@ -66,6 +66,38 @@ def range_end(range_id) -> None:
     _nvtx.rangeEnd(range_id)
 
 
+def _device_range_start(msg: str, stream: int = 0) -> object:
```

Comment:
I'm a bit confused about this one.
Doesn't nvtx allow you to add a device side range natively?

Reply:
If that's the case, then I've certainly missed it. Let me double check.

Reply:
The only device-related APIs I could find are [`nvtxNameCuDeviceX`](https://github.com/NVIDIA/NVTX/blob/v3.1.0/c/include/nvtx3/nvToolsExtCuda.h#L62-L63) which give a custom name for the specific device.

Additionally, there is this discussion [How to use NVTX in device code](https://github.com/NVIDIA/NVTX/issues/83), which is slightly orthogonal, but somewhat related.

Did I miss anything?